.github/workflows/regen_certs.yml

```diff
@@ -0,0 +1,40 @@
+name: Regenerate certifactes
+on:
+  schedule:
+    # Run at 06:00 UTC, every Monday and Thursday.
+    - cron: '0 6 * * 1,4'
+jobs:
+  update:
+    runs-on: ubuntu-18.04
+    steps:
+    - name: Install Go
+      uses: actions/setup-go@v2
+      with:
+        go-version: '^1.13.1'
+    - name: Checkout
+      uses: actions/checkout@v2
+    - name: Run cert_regen.sh
+      # Use a conditional step instead of a conditional job to work around #20700.
+      if: github.repository == 'web-platform-tests/wpt'
+      run: |
+        git config --local user.name "GitHub Action"
+        git config --local user.email "action@github.com"
```

Comment:
Is this necessary? What's the default name and email on GitHub Actions?

Reply:
No longer necessary now that commits are done via the GH action.